src/components/ContributorShowcase/ContributorShowcase.jsx

```diff
@@ -0,0 +1,40 @@
+import { StaticQuery, graphql } from 'gatsby';
+import React from 'react';
+import styles from './ContributorShowcase.module.css';
+
+function getContributors(data) {
```

Comment:
would prefer something more descriptive, e.g.

```suggestion
function renderContributorsFromData(data) {
```